rudder-agent/SOURCES/patches/cfengine/0017-define-classes-when-package-is-up-to-date.patch

```diff
@@ -1,13 +0,0 @@
-diff -ur a/cf-agent/verify_packages.c b/cf-agent/verify_packages.c
```

Comment:
This patch is a duplicate of patch 07-define-classes-when-package-is-up-to-date.patch and has already been removed in the 3.2 branch.